Assets/Tests/InputSystem/CorePerformanceTests.cs

```diff
@@ -1207,5 +1207,58 @@ public IEnumerator Performance_MeasureInputSystemFrameTimeWithProfilerMarkers_Do
             .Run();
     }
 
+    [PrebuildSetup(typeof(ProjectWideActionsBuildSetup))]
+    [PostBuildCleanup(typeof(ProjectWideActionsBuildSetup))]
+    [UnityTest, Performance]
+    [Category("Performance")]
+    public IEnumerator Performance_MeasureInputSystemFrameTimeWithProfilerMarkers_Touch()
+    {
+        var touchscreen = InputSystem.AddDevice<Touchscreen>();
+        EnhancedTouchSupport.Enable();
+
+        var clickAction = InputSystem.actions.FindAction("Click");
+        var pointAction = InputSystem.actions.FindAction("Point");
+
+        int performedCallCount = 0;
+
+        clickAction.performed += context => {
+            performedCallCount++;
+        };
+
+        pointAction.performed += context => {
+            performedCallCount++;
+        };
+
+        using (Measure.ProfilerMarkers(allInputSystemProfilerMarkers))
```

Comment:
I guess changes look ok here but I wonder what scenario we are trying to replicate here? It seems that there is not many touch points per frame here, essentially looks like one touch update per frame. That is fine given a e.g. 30 or 60 Hz touch sensor , but I believe most these days generate higher frequency of touch. It might be meaningful to measure a decently modern touch screen and take the actual achievable frequency. I wouldn't trust just reading specs since the HW frequency might be higher than what you get out of the driver.

Reply:
It be good to outline the intended scenario as clearly as possible in the test description or inline (Seems like @jfreire-unity suggested something similar.